### ISSUE TITLE
[BUG] minecraft:mineable/pickaxe tag disappears while using the mod

### ISSUE THREAD

**Issue by aleats91** [lukebemish/excavated_variants#119]
### What went wrong?

I am attempting to run the modpack Multiblock Madness 2 on Linux. However, while this mod is active, all mineable/pickaxe tags seem to completely disappear, for no apparent reason.
The following line seems to be the source of the problem:
`[18May2023 13:47:47.061] [Worker-Main7/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag minecraft:mineable/pickaxe as it is missing following references: excavated_variants:blackstone_osmium_ore (from dynamic_asset_generator:data)`
All other mineable/pickaxe tags (and other tools, like paxels and drills) stop functioning as well, as they depend on minecraft:mineable/pickaxe to work.

### Platform

Forge

### Version

v0.6.5

### Logs and Screenshots

https://mclo.gs/pQ6IoDn
![exhibit1](https://github.com/lukebemish/ExcavatedVariants/assets/133994290/c60ab002-ceca-4a24-8e40-9d260c40d59c)
![exhibit2](https://github.com/lukebemish/ExcavatedVariants/assets/133994290/7b427e27-7afa-4ced-95e4-6ae070f06168)

**Comment by aleats91** [comment]
I did look at the logs a bit further, and there's a lot more going wrong. Most ore tags are also completely gone, in the most part because of missing references from this mod. However, this seems to be a fairly uncommon error, as very few people seem to have experienced it.

**Comment by lukebemish** [comment]
Hmm. Do you have asset caching for Dynamic Asset Generator turned on?

**Comment by lukebemish** [comment]
You can check that in the config for DynAssetGen

**Comment by aleats91** [comment]
Yes. Both asset caching and data caching are on in the mod config.

**Comment by lukebemish** [comment]
Yeah, that's what I thought. It's probably got old cache entries from before the Excavated Variants config was changed (or removed). I'd check a few things: first, is that cache out of date (try deleting it and letting it regenerate)? Second, is excavated variants even installed, or is this complaining about an out of date cache because it was removed? Third, if Excavated Variants is installed, as it's configs missing? (I think I used a config folder back then, so I'd check the `excavated_variants` folder in the config folder; if it's missing/ empty, then delete the `excavated_variants.json` file for it to regenerate)

**Comment by lukebemish** [comment]
If you've tried deleting the cache and letting it regenerate, and this issue is still occurring, could I have a full mod list and a debug.log file?

**Comment by aleats91** [comment]
Regenerating the Dynamic Asset Gen cache seems to have completely fixed the issue.

**Comment by lukebemish** [comment]
Yep, seems like an out of date cache - though I'm still curious why those blocks don't exist. Does the modpack use a custom configuration for Excavated Variants, or disable it entirely? Regardless, if you have caching turned on, you have to delete the cache every time you update the configuration or update the mod/modpack.

**Comment by aleats91** [comment]
I'm not completely sure why this happened, or why it seems to just affect some people, but I am contacting the modpack maker to see if this issue can be fixed. Notably, the issue where Nether ores spawn in the overworld also happens in this modpack, and seems to have a similar source, if not the exact same.

**Comment by lukebemish** [comment]
Hmm. That seems like it could be partially bad Excavated Variants configs, then, though I'd have to look into it more. For nether ores spawning in the overworld - if that is occurring, some mod is not setting its worldgen up correctly (as in, it's adding nether ore features to all biomes, not just nether biomes, under the assumption that overworld biomes will be missing the stone it would generate in), but this might be fixable by turning off `attempt_ore_generation_insertion` in the config; this will decrease the quality/accuracy of ore replacement slightly but should stop any possibility of an issue like that happening. (this option was removed in newer versions, so I doubt the difference will be too large, though I did improve the generation code so there might still be something noticable)
